src/buspirate.c

```diff
@@ -63,6 +63,8 @@
 #define BP_FLAG_XPARM_CPUFREQ       (1<<5)
 #define BP_FLAG_XPARM_RAWFREQ       (1<<6)
 #define BP_FLAG_NOPAGEDREAD         (1<<7)
+#define BP_FLAG_PUPS                (1<<8)
```

Comment:
May I suggest a slightly different name, e.g. `BP_FLAG_PULLUPS`?
Hint: lookup "Pups" in a German dictionary. :-)